### ISSUE TITLE
Feature request: additional armature/rig choices?

### ISSUE THREAD

**Issue by infinitephantasm** [FlailingFog/KK-Blender-Porter-Pack#277]
i'm unsure how feasible something like this is, so this is more of a curious ask than an actual request.

i've genuinely had quite a bit of trouble using the pre-existing VRM rigs and such for some other things i've wanted to do, i.e. player models in games like Tower Unite, Garry's Mod, etc., so i must ask: how difficult would it be to use their template armatures for something like this project, as an alternative to VRM & rigify?

a rig & some additional SDK documentation for [Tower Unite](https://towerunite.com/sdk/creating-characters) is available. there is substantially less in the case of Garry's Mod, but a [steam workshop guide](https://steamcommunity.com/sharedfiles/filedetails/?id=702924242) with assorted information and files is also available.

again, if none of this is feasible, and all of this would just be too much, then just ignore this and close it. it's mostly just curiosity's sake -- i'm not very experienced with blender, and figuring out programming/python to accomplish something like this is something i've been unable to learn, so something like this would help out lots in the process of making things like this.

**Comment by poisenbery** [comment]
This is just how I do things, but whenever I make a "Feature Request" I've already done some/most of the work associated, or I have a plan to finish said work so that the devs can simply add it to the project.

I've done some feature requests here for combining the normal/detail maps. I provided the node group and did the research myself, so they added it to the project: https://github.com/FlailingFog/KK-Blender-Porter-Pack/issues/166#issuecomment-1157065330 
And symmetrical blush mapping #181 . In both of those cases, I have done all of the work necessary to have it added to the project.

In addition, the guy who did the rigify scripts basically wrote that by himself. 

I have another feature request that is incomplete, but as you can see, I took the initiative to get some of the work done. The only reason it's not added is because it's incomplete. #234 


In my opinion, feature requests SHOULD include the work/research necessary to make it work in the project. If it's just an idea, then you're at the mercy of the dev's free time. 

If you REALLY want this added, do the research and post the code. There's a high chance it will not get done if you simply ask someone else to do it for you.

**Comment by infinitephantasm** [comment]
yeah, i figured that was mostly the case. i unfortunately don't have the spare time or experience to work on stuff like this, or learn the skills required these days, so i figured i'd toss it out as a concept. even if it never happens, it's fine -- mostly just a thought of mine.
